<img src="https://userweb.fct.unl.pt/~jmc.xavier/MSI/iLogos/nsst.jpg" width="400" height="100">

# DEMI | NOVA SST

## Mecânica dos Sólidos I

## Tração e compressão de barras elásticas
## Vigas sujeitas à tracção/compressão. Parte I
## Problema 10 <a name="PIV810"></a>

A barra rígida $ABC$ está ligada a uma articulação em $A$ e é sustentada por dois cabos $BE$ e $DC$, cada um com um diâmetro de 4 mm, como mostra a figura. Se os cabos tiverem um comportamento elastoplástico perfeito, com uma tensão de cedência $\sigma_C = 530$ MPa e um módulo de elasticidade $E = 210$ GPa, determine:

[a)](#A) A intensidade máxima da carga distribuída uniforme que pode ser aplicada na barra $ABC$ sem provocar o início do escoamento plástico de qualquer um dos cabos.

[b)](#B) O deslocamento do ponto $C$ quando a carga determinada na alínea a) é aplicada.

[c)](#C) A intensidade da carga distribuída que provoca a cedência plástica dos dois cabos e o valor do deslocamento em $C$ quando essa carga é atingida.

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MSI/Notebooks/Au08/Au08_P010_i1.jpg"
width="380" height="280">



## Resolução

#### a) A intensidade máxima da carga distribuída uniforme que pode ser aplicada na barra $ABC$ sem provocar o início do escoamento plástico de qualquer um dos cabos. ([enunciatum](#PIV810))

Considerando a tensão de cedência enunciada e a área de secção transversal dos cabos, pode-se determinar a força máxima que sobre estes pode ser aplicada no limite de elascidade:

\begin{equation*}
F_\mathrm{max} = \sigma_C A
\end{equation*}

In [1]:
import numpy as np

# unidades: mm, N, MPa
dia, SigC  = 4, 530

A = np.pi*dia**2/4
print(f'A = {A:.3f} mm^2')
Fmax = SigC*A
print(f'Fmax = {Fmax:.3f} N')

A = 12.566 mm^2
Fmax = 6660.176 N


##### Equílibrio estático

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MSI/Notebooks/Au08/Au08_P010_i2.jpg"
width="380" height="280">

Considere-se o diagrama de corpo rígido da barra rígida $ABC$ com forças verticais positivas nas secções $B$ e $C$ bem como pelo carregamento distribuido, que para efeitos de cálculo de equilíbrio pode ser substituido por uma força resultante vertical negativa aplicada no centro da barra. O equilíbrio do momento em torno do ponto $A$ vem:

\begin{equation*}
\begin{cases}
\sum F_y = 0 \\
\sum M_A = 0 \\
\end{cases}
~\Leftrightarrow~
\begin{cases}
F_{B} + F_{C} - F_r = 0 \\
0.4 F_{B} + 0.65 F_{B} - 0.4 F_r = 0 \\
\end{cases}
\end{equation*}

##### geometria da deformada da barra rígida $ABC$

\begin{equation*}
\tan\theta = \frac{\delta_B}{0.4} = \frac{\delta_C}{0.65}
\quad\Leftrightarrow\quad
\frac{\delta_C}{\delta_B} = \frac{0.65}{0.4}
\end{equation*}

\begin{equation*}
\delta_C > \delta_B
~\Rightarrow~
F_C > F_B
\quad\therefore\quad
F_C = 6660.2~\textrm{N}
\end{equation*}

pelo que o rácio de deslocamentos ou forças permite escrever ($L,A,E$ são iguais para ambos os cabos):

\begin{equation*}
\begin{aligned}
\frac{\delta_C}{\delta_B} &=
\frac{F_C}{F_B} = \frac{0.65}{0.4} = 1.625 \\
F_C &= 1.625 F_B = 6660.2~\textrm{N}
\quad\wedge\quad
F_B = 4048.6~\textrm{N}
\end{aligned}
\end{equation*}

In [2]:
LAB, LBC = 400, 250
LAC = LAB+LBC

FC = Fmax
print(f'FC = {FC:.3f}  N')
racio = LAC/LAB
print(f'LAB/LAC = {racio:.3f}')
FB = FC/racio
print(f'FB = {FB:.3f} N')

FC = 6660.176  N
LAB/LAC = 1.625
FB = 4098.570 N


Pelo que a força distribuida vem

\begin{equation*}
0.4 F_{B} + 0.65 F_{C} - 0.4 F_r = 0
~\Leftrightarrow~
F_r = 14921 ~\textrm{N}
\end{equation*}

em que $F = F_r/L = 14921 / 0.8 = 18651 $ N/m

In [3]:
LCG = 150
LAG = LAB + LBC + LCG
Fr = (LAB*FB + LAC*FC)/(LAG/2)
print(f'Fr = {Fr:.3f} N')
Fd = Fr/(LAG*1e-3)
print(f'Fd = {Fd:.3f} N/m')

Fr = 14921.357 N
Fd = 18651.696 N/m


#### b) O deslocamento do ponto $C$ quando a carga determinada na alínea a) é aplicada. ([enunciatum](#PIV810))

\begin{equation*}
\delta_C = \frac{FL}{AE} = 2.02 ~\textrm{mm}
\end{equation*}

In [11]:
Lcabo = 800
E = 210e3 # unidade: MPa
deltaC = FC*Lcabo/A/E
print(f'deltaC = {deltaC:.3f} mm')

deltaC = 2.019 mm


#### c) A intensidade da carga distribuída que provoca a cedência plástica dos dois cabos e o valor do deslocamento em $C$ quando essa carga é atingida. ([enunciatum](#PIV810))


O cabo $BE$ inicia plastificação quando $F_B = F_\mathrm{max}$ = 6660 N.

\begin{equation*}
0.4 F_{B} + 0.65 F_{C} - 0.4 F_r = 0
~\Leftrightarrow~
0.4 (F_\mathrm{max}) + 0.65 (F_\mathrm{max}) - 0.4 F_r = 0
\end{equation*}

In [12]:
Fr2 = (LAB*Fmax + LAC*Fmax)/(LAG/2)
print(f'Fr2 = {Fr2:.3f} N')
Fd2 = Fr2/(LAG*1e-3)
print('Fd = {Fd2:.3f} N/m')

Fr2 = 17482.963 N
Fd = {Fd2:.3f} N/m


\begin{equation*}
\delta_C = 1.625 \delta_B = 1.625 (2.019)
\end{equation*}

In [13]:
deltaFmac = deltaC
deltaCnew = LAC/LAB*deltaFmac
print(f'deltaCnew = {deltaCnew:.3f} mm')

deltaCnew = 3.281 mm


A componente de deformação plástica vem:

\begin{equation*}
\delta^P = \delta^t - \delta^e
\end{equation*}

In [14]:
deltaP  = deltaCnew - deltaFmac
print(f'deltaPlast = {deltaP:.3f} mm')

deltaPlast = 1.262 mm


In [15]:
deltaP  = deltaCnew - deltaFmac
print(f'deltaPlast = {deltaP:.3f} mm')

deltaPlast = 1.262 mm


\begin{equation*}
\delta_C = 1.625 \delta_B = 1.625 (2.019)
\end{equation*}

In [16]:
deltaFmac = deltaC
deltaCnew = LAC/LAB*deltaFmac
print(f'deltaCnew = {deltaCnew:.3f} mm')

deltaCnew = 3.281 mm


A componente de deformação plástica vem:

\begin{equation*}
\delta^P = \delta^t - \delta^e
\end{equation*}

In [17]:
deltaP  = deltaCnew - deltaFmac
print(f'deltaPlast = {deltaP:.3f} mm')

deltaPlast = 1.262 mm


---

Copyright (c) [Mecânica dos Sólidos I - DEMI - NOVA-SST]

Interactive computing by <a href="https://jupyter.org/" target="_blank"> <span
style="color:#333399"> Jupyter Notebook </span> </a> &nbsp;|&nbsp;Coded by <a href = "mailto: jmc.xavier@fct.unl.pt">José Xavier</a>

Licensed under  <a href="http://creativecommons.org/licenses/by-sa/4.0/"
target="_blank"> <span style="color:#333399;font-size: 20px"> CC BY-SA 4.0  </span></a>
